src/index.js

```diff
@@ -321,7 +321,7 @@ class KadDHT extends EventEmitter {
       waterfall([
         (cb) => utils.convertBuffer(key, cb),
         (id, cb) => {
-          const rtp = this.routingTable.closestPeers(id, c.ALPHA)
```

Comment:
Seeing as this code is the same for every query, maybe it should be part of the Query itself?

Reply:
Agreed